sdk/eventhub/event-hubs/src/batchingReceiver.ts

```diff
@@ -201,13 +234,47 @@ export class BatchingReceiver extends EventHubReceiver {
             sessionError
           );
         }
+        if (receiver && !receiver.isSessionItselfClosed()) {
+          if (!this.isConnecting) {
+            log.error(
+              "[%s] 'session_close' event occurred on the session of receiver '%s' with " +
+                "address '%s' and the sdk did not initiate this. Hence calling detached from the " +
+                "_onSessionClose() handler.",
+              this._context.connectionId,
+              this.name,
+              this.address
+            );
+            await this.detached(sessionError);
+          } else {
+            log.error(
+              "[%s] 'session_close' event occurred on the session of receiver '%s' with " +
+                "address '%s' and the sdk did not initiate this. Moreover the receiver is already " +
+                "re-connecting. Hence not calling detached from the _onSessionClose() handler.",
+              this._context.connectionId,
+              this.name,
+              this.address
+            );
+          }
+        } else {
+          if (this._aborter) {
+            this._aborter.removeEventListener("abort", this._onAbort);
+          }
```

Comment:
Why is the aborter related change needed here?

Reply:
Forgot to remove aborter  code from here.